expected/13.11/extra/bool.out

```diff
@@ -0,0 +1,251 @@
+---------------------------------------------------------------------------
+ Foreign Scan on public."type_BOOLEAN"
+   Output: i, b
+   SQLite query: SELECT `i`, sqlite_fdw_bool(`b`) FROM main."type_BOOLEAN"
+(3 rows)
+
+--Testcase 33:
+EXPLAIN (VERBOSE, COSTS OFF)
+SELECT * FROM "type_BOOLEAN+";
+                                      QUERY PLAN                                      
+--------------------------------------------------------------------------------------
+ Foreign Scan on public."type_BOOLEAN+"
+   Output: i, b, t, l
+   SQLite query: SELECT `i`, sqlite_fdw_bool(`b`), `t`, `l` FROM main."type_BOOLEAN+"
+(3 rows)
+
+--Testcase 34: ERR - invalid text affinity because not ISO:SQL text input
+SELECT * FROM "type_BOOLEAN+";
+ERROR:  SQLite data affinity "text" disallowed for PostgreSQL data type "boolean"
+HINT:  In column "b" expected SQLite affinity "integer", incorrect value = 'x'
+--Testcase 35
+DELETE FROM "type_BOOLEAN" WHERE i = 23;
+--Testcase 36:
+SELECT * FROM "type_BOOLEAN+";
+ i  | b |    t    | l 
+----+---+---------+---
+  1 | t | integer | 1
+  2 | f | integer | 1
+  3 | t | text    | 4
+  4 | f | text    | 5
+  5 | t | text    | 4
+  6 | f | text    | 5
+  7 | t | text    | 3
+  8 | t | text    | 3
+  9 | t | text    | 3
+ 10 | f | text    | 2
+ 11 | f | text    | 2
+ 12 | f | text    | 2
+ 13 | f | text    | 3
+ 14 | f | text    | 3
+ 15 | t | text    | 2
+ 16 | t | text    | 2
+ 17 | t | text    | 1
+ 18 | t | text    | 1
+ 19 | t | text    | 1
+ 20 | t | text    | 1
+ 21 | f | text    | 1
+ 22 | f | text    | 1
+ 24 | f | integer | 1
+ 25 | t | integer | 1
+ 26 |   | null    |  
+(25 rows)
+
+--Testcase 37:
+SELECT * FROM "type_BOOLEAN+" WHERE b IS NULL;
+ i  | b |  t   | l 
+----+---+------+---
+ 26 |   | null |  
+(1 row)
+
+--Testcase 38:
+SELECT * FROM "type_BOOLEAN+" WHERE b IS NOT NULL;
+ i  | b |    t    | l 
+----+---+---------+---
+  1 | t | integer | 1
+  2 | f | integer | 1
+  3 | t | text    | 4
+  4 | f | text    | 5
+  5 | t | text    | 4
+  6 | f | text    | 5
+  7 | t | text    | 3
+  8 | t | text    | 3
+  9 | t | text    | 3
+ 10 | f | text    | 2
+ 11 | f | text    | 2
+ 12 | f | text    | 2
+ 13 | f | text    | 3
+ 14 | f | text    | 3
+ 15 | t | text    | 2
+ 16 | t | text    | 2
+ 17 | t | text    | 1
+ 18 | t | text    | 1
+ 19 | t | text    | 1
+ 20 | t | text    | 1
+ 21 | f | text    | 1
+ 22 | f | text    | 1
+ 24 | f | integer | 1
+ 25 | t | integer | 1
+(24 rows)
+
+--Testcase 39:
+SELECT * FROM "type_BOOLEAN+" WHERE b;
+ i  | b |    t    | l 
+----+---+---------+---
+  1 | t | integer | 1
+  3 | t | text    | 4
+  5 | t | text    | 4
+  7 | t | text    | 3
+  8 | t | text    | 3
+  9 | t | text    | 3
+ 15 | t | text    | 2
+ 16 | t | text    | 2
+ 17 | t | text    | 1
+ 18 | t | text    | 1
+ 19 | t | text    | 1
+ 20 | t | text    | 1
+ 25 | t | integer | 1
+(13 rows)
+
+--Testcase 40:
+SELECT * FROM "type_BOOLEAN+" WHERE NOT b;
```

Comment:
Could you add test for UPDATE/INSERT/DELETE for this table also?

Reply:
This is not store table, @nxhai98 , this is a view - SQLite storage indicator for `type_BOOLEAN` data. All possible `INSERT` yet tested with a binded new values `0` and `1`, `UPDATE` and `DELETE` will be added. Thanks for idea!

Reply:
`UPDATE` and `DELETE` tests added in https://github.com/pgspider/sqlite_fdw/pull/88/commits/2a2261e3bb97b737c260ff2471f3850b4df116e4 . Please confirm TCs, @nxhai98 .

Reply:
Thank you.   
Please help me to resolve this thread.

Reply:
Thank you for updating. Please help me to resolve this thread.